packages/opentelemetry-core/src/trace/NoopTracer.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Tracer,
+  SpanOptions,
+  Span,
+  Propagator,
+  BinaryFormat,
+} from '@opentelemetry/types';
+import { NOOP_HTTP_TEXT_FORMAT } from '../context/propagation/NoopHttpTextFormat';
+import { NOOP_BINARY_FORMAT } from '../context/propagation/NoopBinaryFormat';
+import { NoopSpan } from './NoopSpan';
+
+export const NOOP_SPAN = new NoopSpan({ traceId: '', spanId: '' });
```

Comment:
Java uses specific values for invalid Span/Context:
https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/trace/SpanContext.java#L34
https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/trace/TraceId.java#L36
https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/trace/SpanId.java#L36

I think it would make sense to match their behavior.

Reply:
Could we accomplish a similar thing by having say `exprot const INVALID_TRACE_ID = ''` and `export const INVALID_SPAN_ID = ''` in some authoritative place, maybe in the types package?

Reply:
Sam thing is used in other PR : https://github.com/open-telemetry/opentelemetry-js/pull/102/files#diff-08ac79b280d2466a4fa3ffa58f1caed6R21

Will rebase, which ever is approved later.